frontend/src/metabase/dashboard/actions/save.js

```diff
@@ -121,10 +122,15 @@ export const updateDashboardAndCards = createThunkAction(
         dispatch(saveCardsAndTabs(updatedCardsAndTabs));
       });
 
-      // Make two parallel requests: one to update the dashboard and another for the dashcards and tabs
+      // Make two parallel requests: one to update the dashboard (without dashcards and tabs)
+      // and another for the dashcards and tabs
+      const onlyDashboard = produce(dashboard, d => {
+        delete d.dashcards;
+        delete d.tabs;
+      });
```

Comment:
The reason I had to do this is because the FE was sending dashcards and tabs with `PUT /api/dashboard/:id` when it shouldn't have. It didn't break anything before because the implementation of the API actually didn't care that there were extra keys in the body. But now it does expect that those keys are not there for this to work.